src/UKHO.ERPFacade.Monitoring.WebJob/Program.cs

```diff
@@ -112,6 +113,7 @@ private static void ConfigureServices(IServiceCollection serviceCollection, ICon
 
                 if (eventHubConfig != null && !string.IsNullOrWhiteSpace(eventHubConfig.ConnectionString))
                 {
+                    Console.WriteLine($"eventHubConfig: {eventHubConfig.ConnectionString}");
```

Comment:
Is it required to use Console.WriteLine ? No need to print KeyValutUri, Event Hub ConnectionString and Storage Connection String. Also remove the commented code.

Reply:
Removed